### DISCUSSION TITLE
Not able to use app store connect api for pilot builds

### DISCUSSION THREAD

**Discussion by SumanDas94** [fastlane/fastlane#20638] [General]
Unable to use api_key / api_key_path flags for `bundle exec fastlane pilot builds` . While this https://github.com/fastlane/fastlane/issues/18650 says to use Apple ID + password, the official documentation for pilot says that app store connect api will work for pilot https://docs.fastlane.tools/actions/pilot/. I plan to run it on CI so would want to avoid AppleID + Password and 2FA alternate CI mechanisms

Getting the following error 
```bundler: failed to load command: fastlane (/usr/local/bin/fastlane)
/Library/Ruby/Gems/2.6.0/gems/fastlane-2.209.1/spaceship/lib/spaceship/connect_api/api_client.rb:204:in `handle_response': [!] The URL path is not valid - The relationship 'buildDeliveries' does not exist (Spaceship::UnexpectedResponse)
 ```

@max-ott @tremblay can you help with this ?

I am on fastlane-2.209.1

**Comment by tremblay** [comment]
I have no idea why you tagged me on this to be honest

**Reply by SumanDas94** [reply]
Hi @tremblay i had actually tagged you as i saw the last commit on the pilot build_manager.rb https://github.com/fastlane/fastlane/commit/f221c959f595bd88e011454b533cc08788769612 was from you. Apologies if I had tagged the wrong person

**Comment by vladvlasov256** [comment]
Do API keys work for TF nowadays?
